stdlib/public/SDK/UIKit/UIKit.swift

```diff
@@ -295,3 +295,91 @@ extension UIFocusItem {
   }
 }
 #endif
+
+//===----------------------------------------------------------------------===//
+// NSItemProviderReading/Writing support
+//===----------------------------------------------------------------------===//
+
+#if os(iOS)
+
+public protocol _SwiftDragDropItemProvider {
```

Comment:
I'm a bit leery of adding a whole extra protocol like this. Can we just use _ObjectiveCBridgeable instead?